jazz_test-lambda/README.md

```diff
@@ -1 +1,27 @@
-This is a sample NodeJs template project
+## API for Testing Lambda Functions.
+Provides an API for Testing a lambda function and returns the Execution Status
+#### Request format in JSON
+```
+{
+	"functionARN": "arn:aws:lambda:us-east-1:192006145812:function:jazz20180604-test-lambda-4-test-j4-prod:1",
```

Comment:
Can you please make sure that you aren't putting out actual values anywhere? Anonymize the data as you publish PRs.